terraform/pilot/main.tf

```diff
@@ -15,15 +15,20 @@ variable "branch" {}
 
 variable "security_context" { default = "dev" }
 
-variable "aws_region" { default = "us-east-1" }
-variable "ami_type" { default = "hardened" }
+variable "network" { 
+    type = "map"
+    default = {
+        region = "us-east-1" 
+    }
+}
+variable "ami_type" { default = "not-hardened" }
 variable "bastion_subnet_id" {}
 variable "bastion_security_group_id" {}
-variable "bastion_instance_type" {}
+variable "bastion_instance_type" { default = "m3.medium" }
```

Comment:
Not sure. Typically, AWS requires at least medium instance types to allow you to perform security scans, so this is a "sensible" default, but perhaps another instances type makes more sense.